package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "I used what we’ve learned and take a chance to continue practicing DOM manipulation by dynamically rendering a simple restaurant homepage! At the end, we use use JavaScript alone to generate the entire contents of the website!",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Haroonabdulrazaq/Restaurant-page.git"
+  },
+  "author": "Haroon",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Haroonabdulrazaq/Restaurant-page/issues"
+  },
+  "homepage": "https://github.com/Haroonabdulrazaq/Restaurant-page",
+  "devDependencies": {
```

Comment:
## Include all the required dependencies
- [ ] Kindly ensure all necessary packages for our project gets included in the `here` we currently have `webpack`, `webpack-cli`, but we are missing the `webpack-dev-server`.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

Reply:
All dependencies are available and there were added automatically as soon as they were installed